packages/admin/src/brevoTestContacts/BrevoTestContactsGrid.tsx

```diff
@@ -65,9 +79,21 @@ function BrevoTestContactsGridToolbar({ intl, scope }: { intl: IntlShape; scope:
                 </ToolbarItem>
                 <ToolbarFillSpace />
                 <ToolbarActions>
-                    <Button startIcon={<Add />} component={StackLink} pageName="add" payload="add" variant="contained" color="primary">
-                        <FormattedMessage id="cometBrevoModule.brevoTestContact.newContact" defaultMessage="New test contact" />
-                    </Button>
+                    <Tooltip title={disableButton ? tooltipMessage : ""}>
+                        <span>
```

Comment:
Can you use an fragment here instead of the span? 
<> </>

Reply:
Unfortunately not - this is a weird behaviour of the Tooltip, that I already experienced in other projects as well. If only a fragment is used, the Tooltip is not displayed at all.